metagraph/src/common/vectors/sd_vector_disk.hpp

```diff
@@ -0,0 +1,530 @@
+#ifndef __SD_VECTOR_DISK__
+#define __SD_VECTOR_DISK__
+
+#include <type_traits>
+
+#include <sdsl/sd_vector.hpp>
+#include <sdsl/int_vector.hpp>
+#include <sdsl/int_vector_buffer.hpp>
+#include <sdsl/select_support_mcl.hpp>
+#include <sdsl/util.hpp>
+#include <sdsl/iterators.hpp>
+
+// Based on https://github.com/simongog/sdsl-lite/blob/master/include/sdsl/sd_vector.hpp
+
+// FYI: This implementation does NOT support multithreading
```

Comment:
can you write `TODO` here instead of `FYI` so it'll be easier to find parts of the code that can be improved in the future?

Reply:
It's not clear what's the best way to support multithreading. Memory mapping, probably. It naturally supports multithreading and may be even faster.